supplementary/alica_ros1/alica_ros_turtlesim/libalica-turtlesim/src/RotateTurtle.cpp

```diff
@@ -0,0 +1,33 @@
+#include "RotateTurtle.h"
+#include <engine/logging/Logging.h>
+
+#include <memory>
+
+namespace turtlesim
+{
+
+RotateTurtle::RotateTurtle(alica::BehaviourContext& context)
+        : BasicBehaviour(context)
+{
+}
+
+void RotateTurtle::run()
```

Comment:
why can't we have a separate state on rotateTurtle?

Reply:
We could, but I wanted to further reduce the amount of new elements introduced in the initial tutorial by not having any transitions.